### ISSUE TITLE
can't init with PyCharm on MacOS 10.12.3

### ISSUE THREAD

**Issue by zinwalin** [mailpile/Mailpile#1808]
/usr/local/Cellar/python/2.7.12/Frameworks/Python.framework/Versions/2.7/bin/python2.7 /Users/michael/Development/Mailpile/mp
Metadata index not found: /Users/michael/.mailpile/mailpile.idx                
Elapsed: 0.079s (Loaded metadata, 0 messages)                                  
Elapsed: 0.005s (health: We are healthy!)                                      
pile/crypto/gpgi.py", line 581, in common_args
    version = self.version_tuple()
  File "/Users/michael/Development/Mailpile/mailpile/crypto/gpgi.py", line 562, in version_tuple
    vertext = self.version().strip().split()[-1]
  File "/Users/michael/Development/Mailpile/mailpile/crypto/gpgi.py", line 556, in version
    retvals = self.run(["--version"], novercheck=True)
  File "/Users/michael/Development/Mailpile/mailpile/crypto/gpgi.py", line 638, in run
    proc = Popen(args, stdin=PIPE, stdout=PIPE, stderr=PIPE, bufsize=0)
  File "/Users/michael/Development/Mailpile/mailpile/safe_popen.py", line 153, in __init__
    creationflags=creationflags)
  File "/usr/local/Cellar/python/2.7.12/Frameworks/Python.framework/Versions/2.7/lib/python2.7/subprocess.py", line 711, in __init__
    errread, errwrite)
  File "/usr/local/Cellar/python/2.7.12/Frameworks/Python.framework/Versions/2.7/lib/python2.7/subprocess.py", line 1343, in _execute_child
    raise child_exception
OSError: [Errno 2] No such file or directory

interact error: Failed: interact                                               
Elapsed: 0.011s (interact: Failed: interact )                                  

Process finished with exit code 0

**Comment by step21** [comment]
This is not unique to pycharm or os x. I get the same with an independent python install on Windows 7. At first I thought it was because of a left-over profile directory from an earlier install, but it seems it was not.

**Comment by BjarniRunar** [comment]
It's failing to run the gpg command. Mailpile won't work without GnuPG installed. Renaming the issue; this is something we should detect on startup and give a more helpful error message.

**Comment by step21** [comment]
Yeah. I thought Mailpile shipped its own version anyway, but in my case it is actually that gpg is installed, and even in the PATH, however it is not recognized by mailpile. Can I tell it where it is somehow?
